<launch>
<!-- husky robot-->
  <include file="$(find husky_gazebo)/launch/husky_empty_world.launch"></include>

<!-- load button parameters -->
  <node pkg="teleop_twist_joy" name="teleop_twist_joy" type="teleop_node">
    <rosparam command="load" file="$(find teleop_twist_joy)/config/config.yaml"/>
  </node>

<!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy_node" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <param name="autorepeat_rate" value="20" />
  </node>
</launch>